tests/libjlm/opt/test-unroll.cpp

```diff
@@ -5,6 +5,7 @@
 
 #include "test-operation.hpp"
 #include "test-registry.hpp"
+#include "test-types.hpp"
```

Comment:
Do you actually need this include? I do not see that you defined any of the types in the new test case.